google/cloud/documentai_toolbox/wrappers/document.py

```diff
@@ -64,17 +62,19 @@ def _entities_from_shards(
     """
     result = []
     for shard in shards:
-        for documentai_entity in shard.entities:
-            entity = Entity(documentai_entity=documentai_entity)
-            entity.crop_image(shard)
-            result.append(entity)
-            for documentai_prop in documentai_entity.properties:
-                prop = Entity(documentai_entity=documentai_prop)
-                prop.crop_image(shard)
-                result.append(prop)
-
-    if len(result) > 1 and result[0].documentai_entity.id:
-        result.sort(key=lambda x: int(x.documentai_entity.id))
+        shard_index = int(shard.shard_info.shard_index)
```

Comment:
Do we know that `shard.shard_info.shard_index` will always be the same as the index of `shard` as it appears in the list `shards`?

In `export_images` below we have the following code:
```
image = entity.crop_image(
                documentai_document=self.shards[entity.shard_index]
            )
```
which suggests that these two notion of "shard index" are indeed the same.  Can we confirm that to be the case?

Alternatively, from the perspective of the Toolbox library, the "index of a shard as it appears in the list shards" is the one actually used, so perhaps we can simply iterate with `for shard_index, shard in enumerate(shards): ...` here instead.

Reply:
It should be the same, unless there are any missing (which is a bigger issue tbh)

Maybe I should add an exception for missing shards, because that would cause bigger issues down the line